tests/python/pants_test/engine/test_isolated_process.py

```diff
@@ -29,6 +29,76 @@ class ShellCat(Binary):
   def bin_path(self):
     return '/bin/cat'
 
+  def gen_argv(self, snapshots):
+    cat_file_paths = []
+    for s in snapshots:
+      cat_file_paths.extend(f.path for f in s.files)
+    # TODO(cosmicexplorer): ensure everything is a real path?
```

Comment:
If a file is named e.g. `-e` or another of `cat`'s expected options it would parse it as an option, not a file. This is something that should be addressed in general but this TODO can be removed.